earthfile2llb/interpreter.go

```diff
@@ -1350,6 +1353,21 @@ func (i *Interpreter) handleImport(ctx context.Context, cmd spec.Command) error
 	return nil
 }
 
+func (i *Interpreter) handleCache(ctx context.Context, cmd spec.Command) error {
+	if len(cmd.Args) != 1 {
+		return errors.Errorf("invalid number of arguments for CACHE: %s", cmd.Args)
+	}
+	dir := cmd.Args[0]
+	if !path.IsAbs(dir) {
+		dir = strings.TrimLeft(dir, "./")
+		dir = path.Join("/", i.converter.mts.Final.MainImage.Config.WorkingDir, dir)
```

Comment:
Is this the best way to convert a relative path in `WORKDIR` to an absolute path?

Reply:
maybe you can just do `path.Clean(path.Join("/", i.converter.mts.Final.MainImage.Config.WorkingDir, dir))` to avoid the call to `TrimLeft`, as well as account for cases where `dir` starts with `../`.

Reply:
worked great, thanks for the suggestion